test/both/transport/connection-test.js

```diff
@@ -294,6 +294,30 @@ describe('Transport/Connection', function () {
       })
     })
 
+    it('should ignore WINDOW_UPDATE frame after END_STREAM', function (done) {
+      client.request({
+        path: '/hello'
+      }, function (err, stream) {
+        assert(!err)
+
+        stream.resume()
+
+        stream.once('close', function () {
+          client.on('frame', function (frame) {
+            assert(!(frame.type === 'RST' && frame.id === 1))
+          })
```

Comment:
If you could move this assert (306~308) to outside of the `stream.once('close'`, you could send multiple windowUpdateFrame and see it being processed before and confirming that it doesn't get emitted after the close happens.